### DISCUSSION TITLE
Import Existing SnowFlake objects

### DISCUSSION THREAD

**Discussion by chintan-episource** [littleK0i/SnowDDL#21] [Q&A]
Can we import existing objects created manually in snowflake using snowddl.? This will facilitate  transitioning objects already created as manual.

**Comment by littleK0i** [comment]
Please check this answer in another thread: https://github.com/littleK0i/SnowDDL/discussions/8#discussioncomment-2976591

Currently it is possible to convert a small number of object types. I'll improve this feature if I see more interest (your post helps!).
